src/vivarium/framework/values/manager.py

```diff
@@ -138,12 +131,9 @@ def register_attribute_producer(
             pipeline,
             source,
             component,
-            requires_columns,
-            requires_values,
-            requires_streams,
-            required_resources,
-            preferred_combiner,
-            preferred_post_processor,
+            required_resources=required_resources,
```

Comment:
Nit: why are these now kwargs but not the args above?

Reply:
Because the args are in the correct position but then we skip a bunch of optional args (requires_*) so need to add the keyword.